### DISCUSSION TITLE
Add last access date to variables and connections

### DISCUSSION THREAD

**Discussion by alejandrofm** [apache/airflow#35570] [Ideas] [answered]
### Description

The general idea is to add a column on "variables" and in "connections" that shows the last time it was accessed, somehow like AWS Secret Manager

### Use case/motivation

It could be hard to maintain an evergrowing list of variables/connections, in this way, the recently used could be easily distinguishable from the unused ones, this could help to review/delete and keep that data clean and up to date.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by Taragolis** [comment, marked as answer]
> somehow like AWS Secret Manager

You could use [AWS Secret Manager](https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/secrets-backends/aws-secrets-manager.html#aws-secrets-manager-backend) as Alternate Secret Backend for Airflow and monitor access there

**Reply by alejandrofm** [reply]
didn't know that feature, will look into that, thanks!
